src/core/print.ml

```diff
@@ -132,6 +135,13 @@ and pp_term : term pp = fun oc t ->
         else
           out oc "%a ⇒ %a" (pp `Appl) a (pp `Func) c;
         if wrap then out oc ")"
+    | LLet(t,a,u) ->
+        if wrap then out oc "(";
+        let x, u = Bindlib.unbind u in
+        out oc "let %a" pp_tvar x;
+        if !print_domains then out oc ":%a" (pp `Atom) a;
```

Comment:
add space after ":"

Reply:
I think Rodolphe tries to keep things compact, there is no space in general after colons